src/convertToObject.js

```diff
@@ -6,7 +6,25 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesObject = {};
+  const stylesArray = sourceString.split(';').filter(Boolean);
+
+  stylesArray.forEach((style) => {
+    const parts = style.split(':');
+
+    if (parts.length < 2) {
+      return;
+    }
```

Comment:
[CODE PRACTICE] - While it's good that you're checking for the existence of both parts of the style declaration, consider that this check might not be necessary if you use a more robust splitting strategy. For example, you could split on the colon and include a limit to ensure that you only ever get two parts, even if the value contains colons.